client/src/js/SM/ReviewsImport.js

```diff
@@ -2701,20 +2701,29 @@ async function showImportResultFiles(collectionId) {
             }
 
             async function importReviewArray(collectionId, assetId, reviewArray) {
-                let url = `${STIGMAN.Env.apiBase}/collections/${collectionId}/reviews/${assetId}`
-                let result = await Ext.Ajax.requestPromise({
-                    url: url,
-                    method: 'POST',
-                    headers: { 'Content-Type': 'application/json;charset=utf-8' },
-                    jsonData: reviewArray
-                })
-                const apiReviews = JSON.parse(result.response.responseText)
-                return {
-                    inserted: apiReviews.affected.inserted,
-                    updated: apiReviews.affected.updated,
-                    rejected: apiReviews.rejected
+                if (reviewArray.length < 1) { //Don't try to POST reviews if the review Array is empty.
```

Comment:
Would suggest just going with:

```
if (!reviewArray.length) {
  ...
```
this will handle situations where reviewArray is not provided (undefined) or is not an array.

Reply:
done!